src/main/java/net/dv8tion/jda/api/entities/Guild.java

```diff
@@ -296,16 +296,16 @@ default RestAction<Void> deleteCommandById(long commandId)
      * @throws IllegalArgumentException
      *         If the id is not a valid snowflake
      *
-     * @return {@link RestAction} - Type: {@link List} of {@link CommandPrivilege}
+     * @return {@link RestAction} - Type: {@link List} of {@link IntegrationPrivilege}
      */
     @Nonnull
     @CheckReturnValue
-    RestAction<List<CommandPrivilege>> retrieveCommandPrivilegesById(@Nonnull String commandId);
+    RestAction<List<IntegrationPrivilege>> retrieveCommandPrivilegesById(@Nonnull String commandId);
```

Comment:
This can also now return privileges for applications, by passing an application id afaik

Reply:
Would you prefer renaming the method?

Reply:
Something like `retrieveIntegrationPrivilegesById(String targetId)` with an explanation that you can use app id as well